### PR TITLE
Return date of cached response

### PR DESCRIPTION
**Pull request by libbkmz**
This is a follow-up for PR #20 which can be automatically merged now.

### CODE REVIEW

File: tests/test_cache.py [hunk 1]
```diff
@@ -483,6 +483,15 @@ def test_cache_unpickle_errors(self):
...
         self.assertTrue(resp.from_cache)
         self.assertEquals(resp.json()["args"]["q"], "1")
+        
+    def test_cache_date(self):
+        url = httpbin('get')
+        response1 = self.s.get(url)
+        response2 = self.s.get(url)
+        response3 = self.s.get(url)
+        self.assertEqual(response1.cache_date, None)
+        self.assertTrue(isinstance(response2.cache_date, datetime.datetime))
```
Comment (JWCook) [inline tests/test_cache.py:493]: Looks like this needs to be updated to:
```python
self.assertTrue(isinstance(response2.cache_date, datetime.datetime))
```

**Comment by JWCook** [comment]
Great, thanks @libbkmz! There's just one failing unit test, but it looks like an easy fix.

**Comment by libbkmz** [comment]
I've fixed this easy case, but looks like Travis don't want to run Python 2.7 somehow =\ it can't import mock for test_cache.py file

**Comment by JWCook** [comment]
No worries, I'm going to merge PR #173 after yours, which drops python 2 support.

**Comment by JWCook** [comment]
Done. The Travis CI job is green again! https://travis-ci.org/github/reclosedev/requests-cache/builds/760645494

**Pull request merged**